code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -29,6 +31,18 @@
 	results = list(/datum/reagent/impedrezene = 2)
 	required_reagents = list(/datum/reagent/mercury = 1, /datum/reagent/oxygen = 1, /datum/reagent/consumable/sugar = 1)
 
+/datum/chemical_reaction/concentrated_bz
+	name = "Concentrated BZ"
+	id = "Concentrated BZ"
+	results = list(/datum/reagent/concentrated_bz = 10)
+	required_reagents = list(/datum/reagent/toxin/plasma = 40, /datum/reagent/nitrous_oxide = 10)
```

Comment:
So you can only make this in multiples of 10?

Reply:
It's intended, in order to make it harder to mass oroduce, to prevent people from making a crap tonne of it and putting it in a smoke machine or whatever to get a valids radar of sorts.